content/applications/sales/point_of_sale/shop/posreceipts.rst

```diff
@@ -0,0 +1,37 @@
+========
+Receipts
+========
+
+Configuration
+=============
+
+Set up the receipts, by going to :menuselection:`Point of Sale --> Configuration --> Point of Sale`,
+selecting a POS, and scrolling down to the :guilabel:`Bills & Receipts` section.
+
+To **customize** the :guilabel:`Header & Footer`, activate the setting of the same name. Doing so
+enables fields to fill in with the information you want to be printed on the header and/or the
+footer. To **print receipts** automatically once the payment is registered, enable the
+:guilabel:`Automatic Receipt Printing` setting.
+
+.. image:: receipts/receipt.png
+   :align: center
+   :alt: pos receipt
+
+.. seealso::
+   - :doc:`../restaurant/bill_printing`
+   - :doc:`../restaurant/split`
+
+Reprint a receipt
+=================
+
+From the POS interface, click :guilabel:`Orders`, open the dropdown selection menu next to the
```

Comment:
```suggestion
From the POS interface, click :guilabel:`Orders`, open the dropdown selection menu next to the
```
Where is this ? Be it the dashboard or the POS interface, I cannot find any dropdown selection menu. On the *dashboard*, I can do :menuselection:`Orders --> ??`, and on the POS *interface*, I can only click :guilabel:`Quotation/Order`

Reply:
explained on discord :)